internal/server/server_helper.go

```diff
@@ -0,0 +1,28 @@
+package server
```

Comment:
if we are going to introduce behaviour in the server, it should be done through middlewares. 

although this is something that should be done in the controller